central/sensor/service/connection/connection_impl.go

```diff
@@ -366,6 +368,47 @@ func (c *sensorConnection) processIssueLocalScannerCertsRequest(ctx context.Cont
 	return nil
 }
 
+func (c *sensorConnection) processIssueSecuredClusterCertsRequest(ctx context.Context, request *central.IssueSecuredClusterCertsRequest) error {
+	requestID := request.GetRequestId()
+	clusterID := c.clusterID
+	namespace := c.sensorHello.GetDeploymentIdentification().GetAppNamespace()
+	errMsg := fmt.Sprintf("issuing Sensor certificates for request ID %q, cluster ID %q and namespace %q",
```

Comment:
```suggestion
	errMsg := fmt.Sprintf("issuing Secured Cluster certificates for request ID %q, cluster ID %q and namespace %q",
```

... to make it more precise for customers seeing this error in the logs?